trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-bank/index.ts

```diff
@@ -0,0 +1,28 @@
+import { Request, Response } from 'express';
+import { getFormattedReportPeriodWithLongMonth } from '@ukef/dtfs2-common';
+import api from '../../../api';
+import { asUserSession } from '../../../helpers/express-session';
+import { PRIMARY_NAVIGATION_KEYS } from '../../../constants';
+
+export const getUtilisationReportReconciliationByReportId = async (req: Request, res: Response) => {
+  const { userToken, user } = asUserSession(req.session);
+  const { reportId } = req.params;
+
+  try {
+    const utilisationReportReconciliation = await api.getUtilisationReportReconciliationById(reportId, userToken);
+
+    const formattedReportPeriod = getFormattedReportPeriodWithLongMonth(utilisationReportReconciliation.reportPeriod);
+
+    // TODO FN-1615 update when feeRecord information is displayed on the page
+    return res.render('utilisation-reports/utilisation-report-reconciliation-for-bank.njk', {
+      user,
+      activePrimaryNavigation: PRIMARY_NAVIGATION_KEYS.UTILISATION_REPORTS,
+      bank: utilisationReportReconciliation.bank,
+      formattedReportPeriod,
+      // feeRecords: utilisationReportReconciliation.feeRecords,
```

Comment:
Again, please can we just not have the TODO and the commented out feeRecords property